src/blackmetal/CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+cmake_minimum_required(VERSION 3.8)
+project(blackmetal)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+	add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(geometry_msgs REQUIRED)
+
+add_subdirectory(src/communication)
+
+# Main Program
+add_executable(blackmetal src/blackmetal.cpp)
+target_link_libraries(blackmetal
+	bm_communication
+)
+target_include_directories(
+	blackmetal
+PUBLIC
+	$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+	$<INSTALL_INTERFACE:include>
+)
+
+target_compile_features(
+	blackmetal
+PUBLIC
+	c_std_99
+	cxx_std_17
```

Comment:
toto asi nepotrebujes ked setujes 17 v set(CMAKE_CXX_STANDARD 17)

Reply:
To su defaultne konfiguracie co vyhodi ```ros2 pkg create <package_name> --node-name <node_name>```

Reply:
Mozem vvyskusat ci to pojde aj bez toho popripade to potom hodim naspat

Reply:
eb022081df89aecde376f6230586632b2d02ae3e slo to tak to necham tak ako si navrhol